iOS_SDK/OneSignalSDK/UnitTests/UnitTests.m

```diff
@@ -1813,13 +1813,13 @@ - (void)testPushNotificationToken {
     
     XCTAssertNil(observer->last.from.userId);
     XCTAssertNil(observer->last.to.userId);
-    XCTAssertFalse(observer->last.to.subscribed);
+    XCTAssertFalse(observer->last.to.isSubscribed);
     
     [OneSignal disablePush:false];
     [UnitTestCommonMethods runBackgroundThreads];
     
-    XCTAssertFalse(observer->last.from.userSubscriptionSetting);
-    XCTAssertFalse(observer->last.to.userSubscriptionSetting);
+    XCTAssertFalse(observer->last.from.isPushDisabled);
```

Comment:
shoudnt this swap to AssertTrue?

Reply:
This unit test has actually never worked. observer->last is always nil (even in master) so all of these asserts are meaningless. I am working on a way to fix it